provider-example/src/sign.rs

```diff
@@ -0,0 +1 @@
+pub mod ecdsa;
```

Comment:
It seems a bit overkill to make a directory for the `sign` module. Maybe just inline the `ecdsa` mod here?